src/main/java/fm/last/moji/impl/MojiImpl.java

```diff
@@ -72,7 +72,7 @@ public void copyToMogile(File source, MojiFile destination) throws IOException {
       outputStream.flush();
     } finally {
       IOUtils.closeQuietly(inputStream);
-      IOUtils.closeQuietly(outputStream);
+      outputStream.close();;
```

Comment:
We have an old version of _ngnix_, which doesn't support `chunked` requests and response with a `411 Content length required`, when I call `moji.copyToMogile(file, destination)`.

The problem was that neither the exception was thrown nor a useful error log was printed. The issue is related to #21 , which prints
```
411 Content length required
```

where as the current behavior ( including this fix ) is throwing an exception that contains
```
ERR size_verify_error
```